base_layer/wallet/src/output_manager_service/storage/sqlite_db.rs

```diff
@@ -883,11 +883,18 @@ struct NewOutputSql {
     height: i64,
     script_private_key: Vec<u8>,
     script_offset_public_key: Vec<u8>,
+    sender_metadata_signature: String,
 }
 
 impl NewOutputSql {
-    pub fn new(output: DbUnblindedOutput, status: OutputStatus, tx_id: Option<TxId>) -> Self {
-        Self {
+    pub fn new(
+        output: DbUnblindedOutput,
+        status: OutputStatus,
+        tx_id: Option<TxId>,
+    ) -> Result<Self, OutputManagerStorageError> {
+        let sender_metadata_signature = serde_json::to_string(&output.unblinded_output.sender_metadata_signature)
```

Comment:
I see why you did this, but it's probably better to store this as two fields (`sender_metadata_signature_nonce`, `sender_metadata_signature_sig`) in the database and save on the serde hit here